public/css/style.css

```
@@ -1,286 +1,971 @@
+  --color-error: #ef4444;
+  
+  /* Spacing */
+  --spacing-xs: 0.5rem;
+  --spacing-sm: 1rem;
+  --spacing-md: 1.5rem;
+  --spacing-lg: 2rem;
+  --spacing-xl: 3rem;
+  --spacing-2xl: 4rem;
+  
+  /* Typography */
+  --font-family-sans: 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif;
```

Stylelint (17.5.0)

[error] Expected "Roboto" to be "roboto" (value-keyword-case)

(value-keyword-case)